docs/Resources/resources.md

```diff
@@ -3,18 +3,13 @@ Using OpenWorm Resources
 
 This page describes content that has been created by the project for use by the public. Currently we make simulation engines, visualization environments, and data sets available.
 
-Simulation engines
-------------------
-
-### Sibernetic
+For Biologists
```

Comment:
in the For Biologists section requireed?

Reply:
The link is appropriate, but I'd move it into the intro paragraph